### ISSUE TITLE
Dump Jackboots stopped working

### ISSUE THREAD

**Issue by Shahelion** [Ilya3point999K/ThaumicConcilium#18]
I have no idea why, but they suddenly stopped working for me. They still make the player rise up endlessly, but no longer place blocks at all. Still works in other worlds.

**Comment by Ilya3point999K** [comment]
How's that, you had blocks in your arms?

**Comment by Shahelion** [comment]
Yes, tried with several blocks, modded and vanilla. It only stopped working in that one world.

**Comment by Ilya3point999K** [comment]
Paranormal activity I guess

**Comment by Shahelion** [comment]
I'll report back if I manage to reproduce it.

**Comment by Shahelion** [comment]
It works again in that world after a restart, so I'm going to assume this was an isolated incident.

**Comment by RaidhoSketch** [comment]
Piping in - they do not work for us as well, and a restart does nothing.
We thought it was a conflict with BetterPlacement, but after removing it the issue still persisted. Hmmm.

**Comment by Ilya3point999K** [comment]
Would you please link your modpack or modlist?

**Comment by RaidhoSketch** [comment]
Here's a whole Prism instance file: https://gofile.io/d/yknO7u

**Comment by Ilya3point999K** [comment]
What exactly were you doing? Boots didn't work at all, or just stopped working eventually?

**Comment by RaidhoSketch** [comment]
Never worked at all, with a block in hand they just continuously jump without placing blocks
